test/e2e-olm/elasticsearch_metric_test.go

```diff
@@ -218,12 +218,11 @@ func newClusterRole(t *testing.T, client client.Client, name string) (*rbac.Clus
 
 	err := client.Create(context.TODO(), cr)
 	if err != nil {
-		return nil, err
+		t.Log("Unable to create cluser role", "error", err)
```

Comment:
You'll have to use `t.Logf(` to format this better since the testing.T.Log func does not do formatting. This message will log like this:

```
Unable to create cluser role error some long error here
```

This looks more like a runon sentence than a well formatted string :) 
[Example](https://play.golang.org/p/puryLDsjqlI)